tests/provider/eccc/test_api.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018-2021, earthobservations developers.
+# Distributed under the MIT License. See LICENSE for more info.
+import numpy as np
+import pandas as pd
+import pytz
+from pandas._testing import assert_frame_equal
+
+from wetterdienst.provider.eccc.observation import EcccObservationRequest
+
+
+def test_eccc_api_stations():
+    request = EcccObservationRequest(
+        parameter="DAILY",
+        resolution="DAILY",
+        start_date="1990-01-01",
+        end_date="1990-01-02",
+        humanize=True,
+        tidy=True,
+        si_units=False,
+    ).filter_by_station_id(station_id=(14,))
+
+    expected = pd.DataFrame(
+        {
+            "station_id": ["14"],
+            "from_date": [pd.Timestamp("1984-01-01", tz=pytz.UTC)],
+            "to_date": [pd.Timestamp("1996-01-01", tz=pytz.UTC)],
+            "height": [4.0],
+            "latitude": [48.87],
+            "longitude": [-123.28],
+            "name": ["ACTIVE PASS"],
+            "state": ["BRITISH COLUMBIA"],
+        }
+    )
+
+    assert_frame_equal(request.df, expected)
+
+
+def test_eccc_api_values():
+    request = EcccObservationRequest(
+        parameter="DAILY",
+        resolution="DAILY",
+        start_date="1980-01-01",
+        end_date="1980-01-02",
+        humanize=True,
+        tidy=True,
```

Comment:
Does ECCC actually provide `tidy=False` here?

Reply:
Yes, why not?